### PR TITLE
Build: make bra a local dependency

### PR DESCRIPTION
**Pull request by markelog**
Same idea as with the `gosec`, `revive` and etc.
Allows you to execute `make bra` as pinned local dependency instead
as global one. And therefore alleviates dependency installation issues.

README also updated - I was thinking to provide verbose installation instructions
for it. Since with that way we assume make is installed on the
developer system.

But it might be premature right now?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -89,6 +93,16 @@ revive-alerting: scripts/go/bin/revive
 		-formatter stylish \
 		./pkg/services/alerting/...
 
+bra: scripts/go/bin/bra
```
Comment (bergquist) [inline Makefile]: I wonder if we can come up with a better name :)
With this change I dont think we have to care about how the process start. Perhaps we could name it something more in the lines of "start grafana and watch for changes" :)

Comment (markelog) [reply Makefile]: Haha, had exactly the same thoughts! I called it `run` before, but that one is already taken. We can overwrite it though

Comment (bergquist) [reply Makefile]: I think we can replace run with this. I doubt anyone will notice and I dont think two need two ways of starting the server from the makefile.

Comment (markelog) [reply Makefile]: Yes! Done :)

File: README.md [hunk 2]
```diff
@@ -40,31 +39,14 @@ go get github.com/grafana/grafana
...
-
 ### Run and rebuild on source change
 
 #### Backend
 
 To run the backend and rebuild on source change:
 
 ```bash
-$GOPATH/bin/bra run
+make bra
```
Comment (xlson) [inline README.md]:
```suggestion
make run
```

**Review by xlson — APPROVED** [review]
Great work!

**Pull request merged**